src/test/java/api/support/fixtures/ItemMarkWithdrawnFixture.java

```diff
@@ -0,0 +1,34 @@
+package api.support.fixtures;
+
+import static api.support.http.BusinessLogicInterfaceUrls.markWithdrawnUrl;
+import static org.folio.inventory.support.http.client.ResponseHandler.any;
+
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.TimeUnit;
+
+import org.folio.inventory.support.http.client.IndividualResource;
+import org.folio.inventory.support.http.client.OkapiHttpClient;
+import org.folio.inventory.support.http.client.Response;
+
+import lombok.RequiredArgsConstructor;
+import lombok.SneakyThrows;
+
+@RequiredArgsConstructor
+public class ItemMarkWithdrawnFixture {
+  private final OkapiHttpClient okapiClient;
+
+  @SneakyThrows
+  public Response markWithdrawn(IndividualResource item) {
+    return markWithdrawn(item.getId());
+  }
+
+  @SneakyThrows
```

Comment:
Does this convert checked exceptions to unchecked?

Reply:
It actually does not, the checked exception will be propagated in this case. This happens because lombok instrumentation happens at compile time and JVM does not require a throws declaration for checked exceptions at that point.